central/src/components/CreateQuestion.jsx

```diff
@@ -97,12 +96,24 @@ export default function QuestionForm({ updateQuestion, question: initialState, g
       window.alert("Please enter a question");
       return;
     }
-
+    if (question.choices[0].text == null || question.choices[0].text === "") {
```

Comment:
Since checking the string to `null` and empty string is happening in two places, probably better to create a method for it like:

```javascript
const isNullOrEmpty = (str) => {
    return !str || str.length // or str === null || str === ""
}
```